packages/data/src/plugins/persistence/index.js

```diff
@@ -184,19 +182,19 @@ const persistencePlugin = function( registry, pluginOptions ) {
 					initialState = persistedState;
 				}
 
-				options = { ...options, initialState };
+				options = {
+					...options,
+					initialState,
+				};
 			}
 
 			const store = registry.registerStore( reducerKey, options );
 
-			store.dispatch = flow( [
```

Comment:
Can you describe what's changing in this file? Is it related to the prior comment note of "some modules monkey-patching the store object" ?

Reply:
exactly, I made this change prior to the monkey-patching fix. So we could revert it now I think, but ultimately I think this is a good change, so I'm thinking of keeping it now.